### ISSUE TITLE
Unsolveable Sudoku

### ISSUE THREAD

**Issue by SpaghettiLord1010** [kaajjo/Libre-Sudoku#34]
I was just playing my rounds when I came across a peculiar sight, I noticed right from the start that something is wrong as the six and the two are clearly correct, I tried to see what the hints said and they created nothing, after a while I decided to give up to see if I waa mistaken or not, the solution was a completely blank board. ![Screenshot_20230301-062424.png](https://user-images.githubusercontent.com/50876603/222058840-f7d16cca-6464-4fb4-8a6f-0c7e0f54e380.png)

**Comment by Ilithy** [comment]
Hi;

It's really strange, I don't have these problems personally:

![Screenshot_20230301-085428](https://user-images.githubusercontent.com/36798218/222078182-9e46cadb-871c-4b5a-a67d-e51182b1b810.png)

If I give up, and ask him to display the winning grid, I get this:

![Screenshot_20230301-085631](https://user-images.githubusercontent.com/36798218/222078430-bd14a83f-d815-4070-a0d0-ea2ddc1d6d18.png)

Os Android: GrapheneOS (Android 13)
LibreSudoku version: 1.4.1

**Comment by SpaghettiLord1010** [comment]
Ha, that's rather strange then, I have no idea what was wrong here then

**Comment by Ilithy** [comment]
Me neither, maybe the bug appears depending on the level of difficulty chosen, or is it a temporary bug, have you tried again since?

**Comment by SpaghettiLord1010** [comment]
Yeah, it works fine again, strange, difficulty was easy if that helps.

**Comment by kaajjo** [comment]
Yeah, this is very strange. Maybe the solved board didn't initialize properly at the beginning of the game. So I added additional checks to make sure it's properly initialized, hope it helps

**Comment by SpaghettiLord1010** [comment]
Thanks!

**Comment by alexklann** [comment]
Still have this issue on regular basis with the higher difficulties.
Goes up to a point where everything is "blue" (so not red; as in error) and clicking "tip" crashes the entire app.

<img width="256" height="512" alt="Image" src="https://github.com/user-attachments/assets/745c4675-75a8-415d-b6c8-c9eab4951433" />
